client/extensions/zoninator/components/settings/zone/posts-list.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PostCard from './post-card';
+import RecentPostsDropdown from './recent-posts-dropdown';
+import SearchAutocomplete from './../../search-autocomplete';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import SortableList from 'components/forms/sortable-list';
+
+class PostsList extends Component {
+
+	addPost = ( { push } ) => {
+		return ( post ) => push( post );
+	}
+
+	removePost = ( { remove } ) => {
+		return ( idx ) => remove( idx );
+	}
+
+	changePostOrder = ( { move } ) => {
+		return ( newOrder ) => {
+			if ( newOrder.length < 2 ) {
+				return;
+			}
+
+			// This loop attempts to find to which index in the array has been moved
+			// by making the following assumptions:
+			// Moved forward: newOrder[ from ] < from.
+			// Moved backward by less than one position: same as moving the next item forward.
+			// Moved backward by more than one position: newOrder[ from ] > from + 1.
+			let from = 0;
+
+			while ( ! ( newOrder[ from ] < from || newOrder[ from ] > from + 1 ) ) {
+				from += 1;
+			}
+
+			move( from, newOrder[ from ] );
+		};
+	}
+
+	render() {
+		const { fields, translate } = this.props;
+		const posts = fields.getAll() || [];
+		const showPosts = posts.length > 0;
+
+		return (
+			<div>
+				<FormFieldset>
+					<FormLabel>
+						{ translate( 'Add content' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Add Content' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/zoninator-edit-zone)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).